### ISSUE TITLE
[Linux] Appearance errors when opening on a system with a customized terminal.

### ISSUE THREAD

**Issue by h93xV2** [zeit/hyper#2182]
- **OS version and name**: Fedora 25
- **Hyper.app version**: 1.4.3
- **Relevant information from devtools**:
     
      bundle.js:47 Unknown OSC code: "10"
      bundle.js:47 Unknown OSC code: "11"
      bundle.js:47 Unknown OSC code: "12"
      bundle.js:47 Unknown OSC code: "13
     
- **The issue is reproducible in vanilla Hyper.app**: `Is Vanilla.`

## Issue

Currently running my OS's default terminal with some modifications. The only one that seems to be causing an issue for Hyper is my installation of [Powerline](https://github.com/powerline/powerline). Colors are showing up fine, but it seems that Powerline uses some symbols that Hyper doesn't recognize.

To elaborate, I have an edited `.bashrc` file which applies stylistic changes to every shell I open, and this includes applying Powerline.

**Screenshots**
![screenshot from 2017-09-07 11-49-03](https://user-images.githubusercontent.com/17226133/30179895-e8374948-93c2-11e7-9d17-af1ee1a1613e.png)
![screenshot from 2017-09-07 11-50-11](https://user-images.githubusercontent.com/17226133/30179902-ec6d6ba0-93c2-11e7-83c2-7b839ae5bf9f.png)

**Comment by chabou** [comment]
Can you attach some screenshots?

**Comment by h93xV2** [comment]
@chabou I uploaded some screenshots to the original issue report.

**Comment by chabou** [comment]
Can you try on `xterm` branch?

**Comment by h93xV2** [comment]
@chabou Sure, I will give that shot.

**Comment by h93xV2** [comment]
@chabou Still giving me the same issue on the `xterm` branch.

**Comment by chabou** [comment]
Can you give your .bashrc related config?
I presume that it works great with other Term application?

**Comment by h93xV2** [comment]
What I have in my `.bashrc` is what's found [here](https://fedoramagazine.org/add-power-terminal-powerline/).

i.e.

    if [ -f `which powerline-daemon` ]; then
      powerline-daemon -q
      POWERLINE_BASH_CONTINUATION=1
      POWERLINE_BASH_SELECT=1
      . /usr/share/powerline/bash/powerline.sh
    fi

**Comment by welnanick** [comment]
You have a powerline patched font installed and are using it correct?

**Comment by h93xV2** [comment]
No, not currently. I will look into what happens with a patched font.

**Comment by welnanick** [comment]
The instructions for powerline state: 

> Powerline uses several special glyphs to get the arrow effect and some custom symbols for developers. This requires having either a symbol font or a patched font installed in the system. The used application (e.g. terminal emulator) must also either be configured to use patched fonts (in some cases even support it because custom glyphs live in private use area which some applications reserve for themselves) or support fontconfig for powerline to work properly with powerline-specific glyphs.

I think your problem is stemming from an improper set-up of powerline, and not hyper. That is, unless it used to work and suddenly stopped. Try downloading and installing a powerline patched font and telling hyper to use the patched font in your `.hyper.js` file.

**Comment by h93xV2** [comment]
@welnanick Sorry for the delayed reply. After reading your comment and the Powerline documentation, I installed a patched font and was able to get the symbols to display. At this point, I think it best to close this issue that I raised (you were right, it was an incorrectly configured `.hyper.js` file). However, I want to point out that I now have a problem identical to yours where the glyphs are not being correctly displayed.
